namespace cpp dsn

struct end_point
{
	1: i32    ip;
	2: i16    port;
	3: string name;
}

// place holder
struct blob
{
}
